setup.py

```diff
@@ -24,6 +24,7 @@ def get_version(rel_path):
     version=get_version('azure_acquire/__init__.py'),
     packages=find_packages(),
     platforms=['mac', 'unix'],
+    install_requires=['click', 'opencv-python', 'numpy', 'matplotlib', 'jupyter', 'pyk4a', 'jupyter'],
```

Comment:
Jupyter is written twice.

Reply:
Also, jupyter isn't used in the codebase - remove it

Reply:
matplotlib isn't used in the codebase either - remove it

Reply:
i removed matplotlib but i want to keep jupyter in case users what to use jupyter notebook